src/session_pool/session.js

```diff
@@ -76,7 +79,7 @@ export class Session {
             sessionPool,
         } = options;
 
-        const { expiresAt = new Date(Date.now() + (maxAgeSecs * 1000)) } = options;
+        const { expiresAt = this._getDefaultCookieExpirationDate() } = options;
```

Comment:
Just noticed, this ignores the `maxAgeSecs` param and works only with the default. We need to pass it down.